src/command.js

```diff
@@ -5,12 +5,13 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-import { MATCH, RECORD } from './constants';
+import { MATCH, RECORD, CLEAN_SCREENSHOTS } from './constants';
 
 const screenshotsFolder = Cypress.config('screenshotsFolder');
 const updateSnapshots = Cypress.env('updateSnapshots') || false;
 const failOnSnapshotDiff =
-  typeof Cypress.env('failOnSnapshotDiff') === 'undefined';
+  typeof Cypress.env('failOnSnapshotDiff') === 'undefined' ||
+  Cypress.env('failOnSnapshotDiff');
```

Comment:
Does this return a Boolean or a string that says `true`?

Reply:
Very good point.

Reply:
I think [this](https://github.com/cypress-io/cypress/blob/a7dfda986531f9176468de4156e3f1215869c342/packages/server/lib/util/coerce.js) is getting applied to `--env` args [by this](https://github.com/cypress-io/cypress/blob/a7dfda986531f9176468de4156e3f1215869c342/packages/server/lib/util/args.js#L184).

Reply:
It seems to [work as expected](https://repl.it/repls/GuiltyScarceComputer).